aws/data_source_aws_network_acls.go

```diff
@@ -0,0 +1,88 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func dataSourceAwsNetworkAcls() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceAwsNetworkAclsRead,
+		Schema: map[string]*schema.Schema{
+			"filter": ec2CustomFiltersSchema(),
+
+			"tags": tagsSchemaComputed(),
+
+			"vpc_id": {
+				Type:     schema.TypeString,
+				Required: true,
```

Comment:
This parameter does not look required for the API call and its valid to want to search for Network ACLs across multiple VPCs, so it should be made optional or removed (in preference of `filter`): https://docs.aws.amazon.com/sdk-for-go/api/service/ec2/#DescribeNetworkAclsInput